strawberryfields/apps/sample.py

```diff
@@ -110,6 +107,7 @@
 import numpy as np
 
 import strawberryfields as sf
+from strawberryfields.apps.qchem.vibronic import sample as vibronic  # pylint: disable=unused-import
```

Comment:
Do we need this?

Reply:
This is so users can go
```python
from strawberryfields.apps import sample

sample.vibronic()
```
This is how we had it before, and it might be nice to keep it that way since we also show this usage in the paper.